pkg/server/controllers.go

```diff
@@ -693,6 +695,55 @@ func (s *Server) installAPIBindingController(ctx context.Context, config *rest.C
 	})
 }
 
+func (s *Server) installAPIBinderController(ctx context.Context, config *rest.Config, server *genericapiserver.GenericAPIServer) error {
+	controllerName := "kcp-apibinder-controller"
+	config = rest.CopyConfig(config)
+	// Do we need a round tripper here?
```

Comment:
Yes, we do need the multi-cluster round-tripper as even under the VW prefix you'll be adding cluster-specific bits.